Docs/readthedocsupdate (#89)
SuperDARN/pydarn: docs/user/hardware.md

> <!--Copyright (C) SuerDARN Canada, University of Saskatchewan 

Author: ```suggestion
<!--Copyright (C) SuperDARN Canada, University of Saskatchewan 
```

!!! Warning
    The hardware information obtained via this class contains most recent updates to the hardware file as it does not take a specific date as an input. To get specific hardware information, please use `read_hdw_file`.

# Updating Radar and Hardware Information

> Radar hardware is infrequently updated or changed. To update your hardware files without updating or reinstalling pyDARN use the following command:

Reviewer: Hmmm, I feel this is an awkward sentence might be just me... What do you think?

Author: instead of infrequently, maybe occasionally?

Reviewer: ```suggestion
pyDARN does not release new versions based on hardware file changes. Because hardware files change infrequently you can update the hardware files in pyDARN by using the following command:
```

Better?

Author: Yup!

``` python
import pydarn
pydarn.get_hdw_files()
```

> This should also replace any missing files.

Reviewer: ```suggestion 
This should also replace any missing files.

# Troubleshooting

If you get an error on importing pyDARN regarding hardware files or the `get_hdw_files()` is not getting the required files to run pyDARN please create an [issue](https://github.com/SuperDARN/pydarn/issues) detailing your computer specifications, commands used to get the error, and output of the error. 
```
